examples/react-app/src/components/SceneViewer.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useCallback } from 'react';
+
+import { SceneViewer as SceneViewerComp } from '@iot-app-kit/scene-composer';
+import { dataSource } from '../dataSource';
+import { sceneId, componentTypeQueries, viewport, entityQueries, dataBindingTemplate } from '../configs';
+
+const sceneLoader = dataSource.s3SceneLoader(sceneId);
+
+const queries = [
+  ...componentTypeQueries.map((q) => dataSource.query.timeSeriesData(q)),
+  ...entityQueries.map((q) => dataSource.query.timeSeriesData(q))
+];
+
+const SceneViewer = () => {
+
+  const onSelectionChanged = useCallback((e) => {
+    console.log('onSelectionChanged event fired with data: ', e)
+  }, []);
+
+  const onWidgetClick = useCallback((e) => {
+    console.log('onWidgetClick event fired with data: ', e)
+  }, []);
+
+  return (
+    <div className="SceneViewer" >
```

Comment:
is there a reason the Scenviewer component requires a div wrapper? why not just have it built into the component exported from scene composer package?

Reply:
by default, Scenviewer takes the full screen. this div provides a style to it to make it fit in it's spot in the sample app